### ISSUE TITLE
[Feature Request] The screen always flashes white before loading

### ISSUE THREAD

**Issue by yozaam** [darkreader/darkreader#3036]
**Is your feature request related to a problem? Please describe.**
I click on a new page and for a second the screen is bright white, it is a sudden flash and I searched the issues page but didn't find it so sorry if this is a duplicate

**Describe the solution you'd like**
We should let the add-on display a black screen while any window. loading and when the dom is ready, we should first change the css to dark and then make the html.style.display ? 

**Additional context**
I am not sure about the exact context or if this is possible, but thanks for the tool i love using it :D

**Comment by Gusted** [comment]
https://github.com/darkreader/darkreader/issues?q=is%3Aissue+white+flash+is%3Aopen

TL;DR wait for browsers go fix this but so far I see they haven't any priority
